### ISSUE TITLE
Associative array - key value pairs.

### ISSUE THREAD

**Issue by lordspace** [LeaVerou/awesomplete#16809]
is it possible to have key value pairs?

```
<select name="country_code">
   <option value="ca">Canada</option>
</select>
```

**Comment by haroldiedema** [comment]
I was wondering this as well! 
Solved it with a "dirty" workaround by comparing the user-defined value with a value of my array.

:+1:

**Comment by herregroen** [comment]
#16774 implements this functionality. It's been open a while though but you could use it directly from source if you need it.

**Comment by vlazar** [comment]
**To everyone interested in differentiating value from displayed text, aka key/value feature, aka array of objects feature.**

The code is ready and it's almost 100% backward compatible. To ensure everything working fine for you we need your help with testing it in the wild before it's released. See this PR: [Separate label/value for each suggestion in list](https://github.com/LeaVerou/awesomplete/pull/16852)

**Comment by vlazar** [comment]
Separate `label/value` is landed #16866
It is also possible to initialize list with separate `label/value` via `<datalist>` or `<ul>`.

Read about different `label` and `value` at the end of [Basic usage](http://leaverou.github.io/awesomplete/#basic-usage) section and about new `data` method at the end of [Extend](http://leaverou.github.io/awesomplete/#extensibility) section.
